src/detect-requires.c

```diff
@@ -20,6 +20,10 @@
 #include "detect-engine.h"
 #include "rust.h"
 
+/* Set to true if unknown requirements should be ingored. In Suricata
+ * 8, unknown requirements are treated as unsatisfied requirements. */
```

Comment:
ah I think this answers my question. So do we need a warning/doc reference about this being 7 only behavior?